docs/01-Headless/04-Transitions.md

```diff
@@ -0,0 +1,51 @@
+# Transitions
+
+## Overview
+
+Transitions in AgnosUI are widgets allowing to display / hide content smoothly through an animation. At its heart, a transition is a [Directive](03-Directives.md) managing the visibility of inner content by attaching / detaching classes.
```

Comment:
`widgets` ? what do you mean by widget?

Reply:
I mean AgnosUI widget, so controlled by props and returning stores.

Reply:
It is a more global reflection on the term `widget` which is not ubiquitous even if we talk about widgets all the time :D